pkg/sshutil/sshutil.go

```diff
@@ -18,34 +19,74 @@ type PubKey struct {
 	Content  string
 }
 
-// DefaultPubKeys finds ssh public keys from ~/.ssh
+func readPublicKey(f string) (PubKey, error) {
+	entry := PubKey{
+		Filename: f,
+	}
+	content, err := os.ReadFile(f)
+	if err == nil {
+		entry.Content = strings.TrimSpace(string(content))
+	} else {
```

Comment:
Return err if !error.Is(err, os.ErrNorExist)

Reply:
Why? If `ReadFile` returns an error, `content` cannot be used, and we should return the error, regardless of what it is.

It will just be written to the log, and then ignored because `DefaultPubKeys` doesn't return an error. Maybe it should?

Reply:
This is now handled by the caller; any error is reported back to the caller because we are only calling `readPublicKey` on files that are supposed to exist.